client/controllers/curatorEvents.coffee

```diff
@@ -16,6 +16,7 @@ Template.curatorEvents.onCreated ->
     ).map((article)->
       [article.userEventId, article]
     ))
+  Template.curatorEvents.associatedEventIdsToArticles = @associatedEventIdsToArticles
```

Comment:
Here is an example for the reviewFilter variable:
https://github.com/ecohealthalliance/eidr-connect/blob/master/client/views/curatorInbox.jade#L28